### ISSUE TITLE
Loss function details

### ISSUE THREAD

**Issue by ArielleZhang** [OpenDriveLab/HDGT#7]
Hi,

Thanks for the great work! I noticed that in the paper, you set the classification loss weight to be 0.1, is there a reason behind this? Where can I find more information/experiments related to the hyper-parameter tuning of this weight?

Best,

Arielle

**Comment by jiaxiaosong1002** [comment]
Hi, thanks for your interestes. I do not tune this hyper-parameter.  I just follow LaneGCN's setting and I believe it would not influence a lot.
